src/test/java/com/hubspot/jinjava/lib/tag/ForTagTest.java

```diff
@@ -205,6 +205,27 @@ public void testForLoopWithDates() {
     assertEquals(new PyishDate(testDate).toString(), rendered);
   }
 
+  @Test
+  public void testTuplesWithPyList() {
+	  String template = "{% for href, caption in [('index.html', 'Index'), ('downloads.html', 'Downloads'), ('products.html', 'Products')] %}" + 
+	  		"<li><a href=\"{{href|e}}\">{{caption|e}}</a></li>\n" + 
```

Comment:
any reason for running this through the escape filter (e) here?

Reply:
this was mentioned in issue #321, therefore, I used the exact same thing to test the functionality out. there is no specific reason for choosing this template. :sweat_smile:

Reply:
Ok, can you remove anything that's not relevant to the test then? We don't want to be testing the escape filter here.

Reply:
done!